src/components/views/voip/CallView.js

```diff
@@ -79,8 +79,7 @@ module.exports = React.createClass({
         if (call) {
             call.setLocalVideoElement(this.getVideoView().getLocalVideoElement());
             call.setRemoteVideoElement(this.getVideoView().getRemoteVideoElement());
-            // give a separate element for audio stream playback - both for voice calls
-            // and for the voice stream of screen captures
+            // always use a separate element for audio stream playback
```

Comment:
I can imagine looking at this code in a couple of months and thinking "why?" - can you one-liner explain this please?